pmg/models/resources.py

```diff
@@ -231,7 +231,7 @@ class File(db.Model):
     origname = db.Column(db.String(255))
     description = db.Column(db.String(255))
     playtime = db.Column(db.String(10))
-    event_files = db.relationship("EventFile", lazy=True)
+    event_files = db.relationship("EventFile", lazy=True, cascade="all, delete")
```

Comment:
No database migration needed because the database constraint already exists. It is SQLAlchemy that tries to set `file_id` to `NULL` before deleting anything and letting the database handle the cascade.